components/coding/JSQuiz.tsx

```diff
@@ -58,6 +58,51 @@ const JSQuiz = ({}: JSQuizProp) => {
       },
       answer: "variable x;",
     },
+    {
+      displayQuestion: "Which of the following expressions evaluates as True",
```

Comment:
It's good to note all your changes in a PR. Add a bullet point in your PR description saying that you added some more questions to the JS quiz.